app/models/physical_ticket.rb

```diff
@@ -0,0 +1,6 @@
+class PhysicalTicket < ActiveRecord::Base
+	belongs_to :conference
+	belongs_to :user
+	belongs_to :ticket_purchases
+	belongs_to :ticket
```

Comment:
A ticket purchase already belongs to ticket, right? So you don't need this relation with ticket. In case that at some point the ticket is needed we can introduce a thought ticket_purchase relation that do not need extra fields in the database. :wink: